QuickDeployment/QuickDeploymentInternetSetup.md

```diff
@@ -0,0 +1,222 @@
+* If you have purchased your own SSL/TLS certificates.
+
+Essentially, in any scenario where the **fully-qualified domain name (FQDN)** of the QDE server is being modified, you will need to ensure that the "Subject/Common Name" attribute on the SSL certificates matches this FQDN. If you are making any of the above changes, please generate new SSL certificates **_after_** any changes to the FQDN have been completed. 
+
+The `New-SslCertificates.ps1` script in the "C:\choco-setup\files" folder on the QDE VM will:
+* Generate new SSL certificates for your Nexus and CCM Service applications
+* Move these certificates to the appropriate certificate stores
+* Configure the Nexus and CCM services to use these certificates in communication with endpoints
+
+**:exclamation: IMPORTANT:** There is a good chance that the current version of the `New-SslCertificates.ps1` script in the "C:\choco-setup\files" folder on your QDE VM is out-of-date, as we have made numerous updates to it over the past few months. Please use the following command to update your version of this file (to be run in a PowerShell Administrator window):
+
+```powershell
+Invoke-WebRequest -Uri 'https://ch0.co/newssl' -OutFile "$env:SystemDrive\choco-setup\files\New-SslCertificates.ps1"
+```
+
+This script can be utilized in multiple certificate scenarios outlined below. Any time this script is run, please be mindful of the warnings below. 
+
+> :warning: **WARNINGS**
+> * The `New-SslCertificates.ps1` script will appear to prompt for input, and also display some misleading output. This is due to the nature of the Java tooling by which the script interacts with Nexus. Please ignore any spurious prompts and output.
+> * If you provide your own SSL certificate, your **private key** needs to be **exportable** into a Java Keystore. The Nexus application requires this.
+> * **Timezones** and time synchronization is critical when generating SSL Certificates. You'll want to ensure all hosts are utilizing the same NTP time source. Otherwise, there is a potential edge case of generating an SSL Certificate that is not yet valid.
+
+### SSL Certificate Scenarios
+
+There are three possible approaches to acquiring SSL certificates for QDE:
+
+* [Scenario 1: Domain Server Certificates](#scenario-1-domain-server-certificates): This would be the domain server SSL certificate, generated on domain join and trusted by all endpoints on your domain.
+* [Scenario 2: Purchased/Acquired Certificates from CA](#scenario-2-purchasedacquired-certificates-from-ca): This would be an SSL certificate that you purchased or acquired from an external Certificate Authority (CA; e.g. LetsEncrypt).
+* [Scenario 3: Self-Signed SSL Certificates](#scenario-3-self-signed-ssl-certificates): This would be a self-generated SSL certificate, similar to the one that comes with QDE (least-desirable).
+
+Each approach is discussed individually in the sections below.
+
+### Scenario 1: Domain Server Certificates
```

Comment:
Unsure how this works in an over the internet kind of scenario. I may not know.

Reply:
I have clarified where possible in this doc, and included a warning at the bottom of the domain section:

> :warning:**WARNING:** Again, we are assuming you are able to manage certificates in your Active Directory (AD) domain environment. You should have a server with the Active Directory Certificate Services (AD CS) role in place, that can issue and manage digital certificates. A detailed explanation of Certificate Authorities is beyond the scope of this document.